src/utils/dialogmoteUtils.ts

```diff
@@ -1,11 +1,33 @@
 import {
   DialogmoteDTO,
   DialogmoteStatus,
+  SvarType,
 } from "@/data/dialogmote/types/dialogmoteTypes";
 
-export const isAktivtDialogMote = (dialogmote: DialogmoteDTO): boolean => {
+export const isAktivtDialogmote = (dialogmote: DialogmoteDTO): boolean => {
   return (
     dialogmote.status === DialogmoteStatus.INNKALT ||
     dialogmote.status === DialogmoteStatus.NYTT_TID_STED
   );
 };
+
+export const harMottattSvar = (dialogmote: DialogmoteDTO): boolean => {
+  const svarFraArbeidstaker =
+    dialogmote.arbeidstaker.varselList[0]?.svar?.svarType;
+  const svarFraArbeidsgiver =
+    dialogmote.arbeidsgiver.varselList[0]?.svar?.svarType;
+  const svarFraBehandler =
+    dialogmote.behandler?.varselList[0]?.svar[0]?.svarType;
```

Comment:
Kan vi garantere at det alltid er det nullte elementet i lista? Sorterer vi i db?

Reply:
Det skal komme sortert med det nyeste varselet (og svaret for behandler) først fra APIet ja